### PR TITLE
Update reference materials with new logo

### LINKED ISSUE #932: Update reference materials with new logo
The reference documentation uses the old MIT App Inventor logo. It should be updated with the new logo to match the rest of the App Inventor website materials.

### PR DESCRIPTION
**Pull request by conorshipp**
I have updated all of the reference documentation pages to use the new
MIT AI logo.

Fixes #932

### CODE REVIEW

**Comment by ewpatton** [comment]
Thanks @conorshipp for the PR. Can I ask you what the motivation was to change the relative-to-root URLs to use ../../, for example in appinventor/docs/reference/other/extensions.html?

**Comment by conorshipp** [comment]
Previously, the logo didn't actually work on pages in the /other directory. I couldn't really find out why it couldn't find the logo, so I thought that using ../../ was the best solution.

**Comment by ewpatton** [comment]
Were you running the server on your computer, or viewing them in your text/HTML editor? In the latter case, / would be relative to the root directory of your machine, rather than the root directory of the web server. If you run the system using the dev server, you should be able to view the URLs relative to /reference and the logo should appear.

**Comment by conorshipp** [comment]
Yeah, I just opened the HTML files in my browser, the /reference/components logo worked fine. I realised that the problem was because I didn't actually run the server. Do you want me to change the URLs back to what they were originally?

**Comment by kkashi01** [comment]
Why not just use same logo name? This way you won't need to touch so many
files.

**Comment by conorshipp** [comment]
@kkashi01 that's a good point. But even if I did use the same name, I'd still have to edit each file to change the height & width of the image.

**Comment by conorshipp** [comment]
@ewpatton sorry about any confusion; I've changed the file paths back to what they were originally.

**Comment by ewpatton** [comment]
@kkashi01 Replacing the existing image doesn't really work in production due to the fact that the logo has a long TTL for caching (1 year). In this case it is more appropriate to provide a new URL so that users don't continue to see the old logo if they have already visited the reference materials.

File: appinventor/docs/reference/other/extensions.html [hunk 1]
```diff
@@ -414,7 +414,7 @@
       <header class="clearfix" role="banner">
         <div id="top_bar">
           <div id="branding">
-            <div id="logo"><a href="http://appinventor.mit.edu" title="Home page" class="active"><img src="/reference/components/images/logo2.png" alt="Explore MIT App Inventor logo"></a></div>
+            <div id="logo"><a href="http://appinventor.mit.edu" title="Home page" class="active"><img src="/reference/components/images/codi_long.png" alt="Explore MIT App Inventor logo"></a></div>
```
Comment (ewpatton) [inline appinventor/docs/reference/other/extensions.html]: In the other files you modified the width/height of the logo image, but not in this one. You should update the CSS rules to be consistent.

Comment (conorshipp) [reply appinventor/docs/reference/other/extensions.html]: Oops, well spotted!

**Pull request merged**